ui/src/app/views/account-creation-success.tsx

```diff
@@ -48,15 +56,15 @@ export class AccountCreationSuccess
     return <React.Fragment>
       <div style={{padding: '3rem 3rem 0 3rem', marginLeft: '-0.5rem', marginRight: '-0.5'}}>
         <BolderHeader>
-          CONGRATULATIONS!
+          Congratulations!
         </BolderHeader>
         <div>
-          <SmallHeader>
+          <SmallHeader style={{colors: colorWithWhiteness(colors.dark, -1)}}>
```

Comment:
`colors` shouldn't be a property? Also, the color of the text should almost certainly be colors.primary.